File: Heat/ViewModels/Plants/IndexPlantViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Plants
    Public Class IndexPlantViewModel
        Property ID As Integer

        <Display(name:="Nominativo")> _
        Property Name As String

        <Display(name:="Classe impianto")> _
        Property PlantClass As String

        <Display(name:="Tipologia impianto")> _
        Property PlantType As String

        <Display(name:="Codice dell'impianto per la provincia")> _
        Property PlantDistinctCode As String

        <Display(name:="Indirizzo")> _
        Property Address As String



    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Plants/IndexPlantViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Plants
{
    public class IndexPlantViewModel
	{
		public int ID { get; set; }

		[Display(Name = "Nominativo")]
		public string Name { get; set; }

		[Display(Name = "Classe impianto")]
		public string PlantClass { get; set; }

		[Display(Name = "Tipologia impianto")]
		public string PlantType { get; set; }

		[Display(Name = "Codice dell'impianto per la provincia")]
		public string PlantDistinctCode { get; set; }

		[Display(Name = "Indirizzo")]
		public string Address { get; set; }



	}

}
